animalkingdom/src/animalkingdom/Mammals.java

```diff
@@ -0,0 +1,65 @@
+package animalkingdom;
+
+public class Mammals extends AnimalActions{
+	// extends means its extending the class 'AnimalActions'
+	// the class that gets extended MUST BE 'abstract'
+	// in a nutshell this is inhereting information (for a lack of a better word in my vocabulary) from the base class.
+	// see AnimalActions.java line 3
+
+	public Mammals(String name, int year, int food)
+	// This is a class constructor
+	// What this does is take in a single parameter or multiple parameters.
+	// Since Java is such a strictly typed language we need to tell it what type of data is being passed in as a parameter, in this case its a string, int (integer), int (integer)
+	{
+		this.name = name;
+		this.year = year;
+		this.food = food;
+
+		// In Java when we pass parameters we need a way to easily access them.
+		// We need to do this because the field is shadowed by a method or constructor parameter.
+		// for an example on this see the link here > https://docs.oracle.com/javase/tutorial/java/javaOO/thiskey.html
+		// So whats happening here is the data is getting imported from Main.java. passed into here and processed, we then are assigning the data to a new name using the this._data_name_ = newName.
+
+	};
+
+	@Override
+	public String getName(){
+		return name;
+	};
+	// Overrides are a bit complex > https://www.geeksforgeeks.org/overriding-in-java/
+	// Essentially whats happening is @Override is 'overridding' its parent public method.
+	// note how i said public. A private Method or class method cannot, again CANNOT be overridden.
+	// The @Override is followed by a class constructor that returns the value of 'name'
+
+	@Override
+	public String getMoveType(){
+		return "feet";
+	};
+	// Whats happening here is the same as whats happening above.
+	// The Diffrence is this is a pre defined string returning how the mammel moves.
+	// this will be something that will ge used in a System.out.println()
+
+	@Override
+	public String getBreathingType(){
+		return "lungs";
+	};
+	// this one returns how it breaths
+
+	@Override
+	public String getReproductionType(){
+		return "live birth";
+	};
```

Comment:
It seems that you forgot to define a `public abstract String getReproductionType()`  method in your abstract `AnimalActions` class - without doing so, the compiler will run into an error upon compiling due to it getting confused about what method it is supposed to be overriding.